flock_test.go

```diff
@@ -45,36 +45,30 @@ func (t *TestSuite) TearDownTest(c *C) {
 }
 
 func (t *TestSuite) TestNew(c *C) {
-	var f *flock.Flock
```

Comment:
Please revert this. This is a compile time check to ensure we haven't broken the API.